css/style.css

```diff
@@ -9,285 +15,2094 @@ body {
+    top: 70px;
+    width: 250px;
+  }
+}
+
+@media (max-width: 800px) {
+  .total-look__text {
+    font-size: 15px;
+    left: 30px;
+    top: 110px;
+    width: 370px;
+  }
+}
+
+@media (max-width: 800px) {
+  .total-look__button {
+    top: 190px;
+    width: 80%;
+    left: 10%;
+    height: 40px;
+    font-size: 15px;
+  }
+}
+
+@media (max-width: 800px) {
+  .total-look__link {
+    top: 2640px;
+    font-size: 27px;
+    line-height: 30px;
+    width: 250px;
+    margin-left: 50%;
+  }
+}
+
+@media (max-width: 800px) {
+  .total-look__arrow {
+    top: 2700px;
+    width: 18.5%;
+    margin-left: 50%;
+  }
+}
+
+@media (max-width: 630px) {
+  .things {
+    width: 85%;
+    height: 400px;
+  }
+}
+
+@media (max-width: 630px) {
+  .total-look__card {
+    top: 2280px;
+    margin-left: 16%;
+    width: 70%;
+    height: 300px;
+  }
+}
+
+@media (max-width: 630px) {
+  .total-look__button {
+    top: 220px;
+  }
+}
+
+@media (max-width: 600px) {
+  .total-look__text {
+    width: 250px;
+  }
+}
+
+@media (max-width: 520px) {
+  .total-look__button {
+    font-size: 13px;
+  }
+}
+
+@media (max-width: 520px) {
+  .total-look__link {
+    font-size: 18px;
+    width: 170px;
+  }
+}
+
+@media (max-width: 520px) {
+  .bag-title-bold {
+    font-size: 25px;
+  }
+}
+
+@media (max-width: 500px) {
+  .things {
+    height: 350px;
+  }
+}
+
+@media (max-width: 500px) {
+  .total-look__card {
+    top: 2240px;
+    height: 270px;
+  }
+}
+
+@media (max-width: 500px) {
+  .total-look__link {
+    top: 2610px;
+  }
+}
+
+@media (max-width: 500px) {
+  .total-look__arrow {
+    top: 2670px;
+  }
+}
+
+@media (max-width: 500px) {
+  .total-look__button {
+    top: 190px;
+  }
+}
+
+@media (max-width: 430px) {
+  span {
+    font-size: 25px;
+  }
+}
+
+@media (max-width: 430px) {
+  .things {
+    height: 300px;
+  }
+}
+
+@media (max-width: 430px) {
+  .total-look__card {
+    top: 2200px;
+    height: 300px;
+  }
+}
+
+@media (max-width: 430px) {
+  .total-look__text {
+    font-size: 12px;
+    top: 120px;
+    left: 15px;
+  }
+}
+
+@media (max-width: 430px) {
+  .total-look__button {
+    top: 220px;
+  }
+}
+
+@media (max-width: 430px) {
+  .bag-title-bold {
+    font-size: 20px;
+    margin-left: 15px;
+  }
+}
+
+@media (max-width: 430px) {
+  .bag-title {
+    font-size: 14px;
+    margin-left: 15px;
+  }
+}
+
+@media (max-width: 430px) {
+  .total-look__link {
+    top: 2615px;
+  }
+}
+
+@media (max-width: 430px) {
+  .total-look__arrow {
+    top: 2675px;
+  }
+}
+
+@media (max-width: 330px) {
+  .things {
+    width: 95%;
+    height: 40%;
+  }
+}
+
+@media (max-width: 300px) {
+  .things {
+    width: 105%;
+  }
+}
+
+@media (max-width: 326px) {
+  .total-look__card {
+    width: 80%;
+  }
+}
+
+@media (max-width: 326px) {
+  .bag-title-bold {
+    font-size: 17px;
+  }
+}
+
+@media (max-width: 300px) {
+  .total-look__card {
+    width: 90%;
+  }
+}
+
+@media (max-width: 280px) {
+  .total-look__button {
+    font-size: 10px;
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)